### ISSUE TITLE
Problems with U1 - LT3461 welding

### ISSUE THREAD

**Issue by rogerio-acioli** [spenceraxani/CosmicWatch-Desktop-Muon-Detector-v2#19]
Hi, Spencer!

I just assembly and welded  the project during that last week. With the arduino routine compile in the arduino nano, I started to make the tests in the main PCB. Unfortunatelly, at the pins 4 and 5 of the U1 component  a smell of burning appears, like if something was in short, at even with a light in the pin.

Did you have some problem like this or do you know what could be happening? I decided to pick off the component from the PCB and analyse the main PCB weldings again.

Anyway, in the first seconds of test, the OLED worked perfectly, showing the  initial message and the name I gave to the detector.

Sincerelly thanks, 
Rogerio.

**Comment by spenceraxani** [comment]
Hi Rogerio,

If a component is getting at all warm to the touch, there is probably something wrong. Your guess is probably correct, in that there is likely a short somewhere. I haven’t ran into this, but I would start by adding new flux and solder to the components to make sure nothing is bridged. Another good thing to check is that the orientation on the LT3461 and LT1807 are correct. 

Thanks,

Spencer

**Comment by rogerio-acioli** [comment]
Hi, Spencer!

The problem was solved by redoing the welding and changing the inductor L1, which in my case could be with problems.

Now I'm having a new issue: The voltage at the pin HV must be 29.5V? In my detector, it's showing 29.8V. I supposed that could happen due to the LT3461 which was suffering from the warm before, but I don't know if it has more causes.

Sincerely thanks, 
Rogerio.

**Comment by spenceraxani** [comment]
Hi Rogerio,

29.8V will still be fine. Normally there is a +/-0.1V uncertainty with the supply voltage, so you’re not that far off. Anything below 30V will be safe.

Thanks,

Spencer

**Comment by rogerio-acioli** [comment]
Hi Spencer, 

Unfortunatelly, even one year after, I still facing the same issue. I'll try to explain the whole history, hoping to have any help.

First of all, to be clear, I'll call the detectors I builded as MuDec.

The MuDec I, the one we were talking about in this issue a year ago, worked well for few months (~6 months of intermediatally use), but, suddenly, during a test with a very weak radioactive source, the  detector suddenly stopped work and doesnt give anymore voltage at the HV pins.

A year after, I builded the MuDec II, III and IV. As a learned lesson from the first, I understood that might be better to assembly a new one instead of try to realize how to fix the first detector.

Even following the steps by watching the videos or reading the papers, I doesnt had sucess...
MuDec III and IV doesnt showed the right voltage at the HV pins and worst, started to show the same sintoms I had before: smoke and the op amp burning at the pins 4 and 5.

In the other hand, the MuDec II worked perfectl fine, showing exactly the +29.5V at the HV pins, but, in the day after, when I tried to attach the SiPM module, the same sintoms occured.

I saw that more persons are having issues with overheating at the U2, which is close.

I looking for any help! It's very important to me finish this project!

Thanks, 
Rogerio

**Comment by spenceraxani** [comment]
Hi Rogerio,

Overheating of U2 typically happens when it is mounted backwards. Could you take some photos of your PCBs and send them to me? 

Often, if you are running into troubles, it’s best to start of slow and get one thing working at a time. For example, I typically start with building the DC-DC booster circuit, make sure it works, then move onto the rest of the circuit.

Thanks,

Spencer

**Comment by rogerio-acioli** [comment]
Hi, Spencer!

I've made some discoveries with a help from my friend and will describe it below.

 - The overheating component in my detectors actually is the U1;
 - Looks like the project were designed to use the LT3461, with one L1-47uH. Due to the lack of that component, I bought the LT3461A and looking for the datasheet, we realize that we should a 22uH inductor (or 47uH in parallel) for that case;
 - In the video below, we can see the U1 getting over heating. That only happens when I put the SiPM in attach with the rest of the ;
 - In measured at the Vin the +4.6V is good and at the Vout, before put the SiPM Module, I was getting the striaght +29.5V

We're thinking about do a test a part witht the SiPM module, by applying the Vin (+29.5V) from a regulated font and with the osciloscope, see the out from the SiPM module.

Again, thanks for any help! We still don't know what is happening and how to fix it.

Video link for the U1 burning: https://1drv.ms/v/s!AgQqVxNNI4EClapFJvndZg3Vkcb_Ng?e=eRSPMO

Tks, 
Rogério!
![WhatsApp Image 2019-09-21 at 12 35 50](https://user-images.githubusercontent.com/30302736/65559755-53cf2300-df12-11e9-9c39-626ea219cdb5.jpeg)
![WhatsApp Image 2019-09-21 at 14 24 41](https://user-images.githubusercontent.com/30302736/65559756-53cf2300-df12-11e9-8aac-031289e12605.jpeg)
![burning1](https://user-images.githubusercontent.com/30302736/65559838-ac9ebb80-df12-11e9-9f00-f535701aa913.png)
